src/main/resources/Babylon_sv.properties

```diff
@@ -187,11 +188,14 @@ share.all.screens = Dela alla sk\u00E4rmar
 compatibility.mode = Kompatibelt l\u00E4ge f\u00F6r tidigare versioner
 compatibility.mode.enable = Aktivera kompatibilitetsl\u00E4ge!
 compatibility.mode.active = Kompatibiltetsl\u00E4ge aktiverat
-compatibility.mode.info = V\u00E4nligen uppdatera Hj\u00E4lptagare till den senaste versionen f\u00F6r att minimera risker.
+compatibility.mode.info = V\u00E4nligen uppdatera din peer till den senaste versionen f\u00F6r att minimera risker.
```

Comment:
@lammel-hub may I ask you to have a brief look at the Swedish grammar please?

    sv_settings.html
    Babylon_sv.properties

Very appreciated, many thanks in advance!

Reply:
Perfect Swedish! 👌